.github/workflows/dependency-license-check.yml

```diff
@@ -14,6 +14,9 @@ jobs:
     if: (github.actor != 'dependabot[bot]')
```

Comment:
```suggestion
    # this job depends on github.head_ref, only available for pull_request/pull_request_target
    if: github.actor != 'dependabot[bot]' && github.event_name != 'pull_request' && github.event_name != 'pull_request_target'
```

Reply:
@rob-sitro This would allow us to keep running other workflows on push events, while preventing only this action from getting called on push events. Do you think this could be worth doing instead of https://github.com/chronograph-pe/chronograph/pull/6810 etc.?

Reply:
Oh yeah def a better approach. Thanks for pointing this out.